### PR TITLE
data: harshsingh32

### PR DESCRIPTION
**Pull request by harshsingh32**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6087"></a>

### CODE REVIEW

**Review by accodes21 — COMMENTED** [review]
can you restrict the tags everything else is great

File: data/harshsingh32.json [hunk 1]
```diff
@@ -1,29 +1,30 @@
 {
 
-  "name": "Harsh Singh || Web developer and Designer || Looking for 6/12 months internship opportunities 😊 || Please 😊 Hire me",
+  "name": "Harsh Singh"
```
Comment (SaraJaoude) [inline data/harshsingh32.json:3]: add a comma
```suggestion
  "name": "Harsh Singh",
```

File: data/harshsingh32.json [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
 
   "type": "personal",
 
-  "bio": "Open source Enthusiast | Pursuing Bachelor of Technology,CSE, prefinal year| Content Writer | Based in India",
+  "bio": "Open source Enthusiast || Pursuing Bachelor of Technology (CSE), prefinal year || Content Writer || Based in India || Web developer and Designer || Looking for **6/12 months internship opportunities** for **Frontend developer and Devrel** || Please 😊 Hire me",
 
-  "tags": ["Open Source", "Content Writer", "Student", "Web3"],
+
+  "tags": ["Open Source", "Content Writer", "JavaScript", "TypeScript", "HTML5" , "React" , "NextJs", "CSS3"]
```
Comment (SaraJaoude) [inline data/harshsingh32.json:10]: add a comma
```suggestion
  "tags": ["Open Source", "Content Writer", "JavaScript", "TypeScript", "HTML5" , "React" , "NextJs", "CSS3"],
```

File: data/harshsingh32.json [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
 
   "socials": [
 
     { "icon": "FaTwitter", "url": "https://mobile.twitter.com/harsh_hasew7890" },
 
     { "icon": "FaGithub", "url": "https://github.com/harshsingh32" },
 
-    { "icon": "SiHashnode", "url": "https://hashnode.com/@harshsin327" }
+    { "icon": "SiHashnode", "url": "https://hashnode.com/@harshsin327" },
```
Comment (SaraJaoude) [inline data/harshsingh32.json:18]: delete comma
```suggestion
    { "icon": "SiHashnode", "url": "https://hashnode.com/@harshsin327" }
```

Comment (SaraJaoude) [inline data/harshsingh32.json:18]: delete comma
```suggestion
    { "icon": "SiHashnode", "url": "https://hashnode.com/@harshsin327" }
```

File: data/harshsingh32.json [hunk 4]
```diff
@@ -1,29 +1,30 @@
...
+     
 
-  ],
+  ]
```
Comment (SaraJaoude) [inline data/harshsingh32.json]: add a comma
```suggestion
  ],
```

File: data/harshsingh32.json [hunk 5]
```diff
@@ -53,19 +52,20 @@
...
 
-   
+      "url": "https://dev.to/harshsingh32",
 
-  ]
-
- 
+      "icon": "FaDev"
 
+   },
```
Comment (SaraJaoude) [inline data/harshsingh32.json:69]: delete comma
```suggestion
   }
```

**Comment by SaraJaoude** [comment]
> can you restrict the tags everything else is great

Thank you for your suggestion but this is not a blocker and users can have as many tags as they want on their own profiles.

**Comment by SaraJaoude** [comment]
You have conflicts in your file, please resolve. This is usually because your fork is not up to date.  

You may wish to have a look at the following Docs and video: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! :tada:

**Pull request merged**